microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md

```diff
@@ -0,0 +1,209 @@
+
+![The 'Symbol Server' page in DevTools Settings, where you enter your Azure DevOps personal access token.](../../media/2022/03/ado-pat-devtools.png)
+
+See [Securely debug original code by publishing source maps to the Azure Artifacts symbol server](../../../javascript/publish-source-maps-to-azure.md).
+ 
+
+<!-- ====================================================================== -->
+## Microsoft Edge DevTools extension for Visual Studio
+
+<!-- Title: Debug your ASP.NET projects in Visual Studio with the Edge Developer Tools -->
+<!-- Subtitle: Get the Edge Developer Tools extension for VS today! -->
+
+Following up on the success of the Visual Studio Code integration, you can now also embed the Microsoft Edge Developer Tools in Microsoft Visual Studio to debug your ASP.NET projects live.  Download the [Microsoft Edge Developer Tools for Visual Studio](https://aka.ms/edgetools-for-vs) and try it out. 
+
+To try this feature:
+1. Make sure Visual Studio 2022 and the ASP.NET workload are installed.
+1. Set up Web Live Preview as the default Web Forms Designer, as described at [Microsoft Edge Developer Tools for Visual Studio](https://aka.ms/edgetools-for-vs).
+1. In an ASP.NET project, open a webpage of the project in the **Design** window.
+1. In the upper left of the **Design** window, click the **Open Edge DevTools** (![Open Edge DevTools icon.](../../media/2022/03/open-edge-dev-tools-v-s-icon.png)) button:
+
+![An ASP.NET project, opening Edge DevTools.](../../media/2022/03/devtools-extension-v-s-web-forms-designer.png)
+
+Edge DevTools for Visual Studio opens, with the **Elements** tool selected:
+
+![Microsoft Edge Developer Tools for Visual Studio: Elements tool of DevTools.](../../media/2022/03/devtools-extension-visual-studio-elements.png)
+
+The **Network** tool is also open by default:
+
+![Microsoft Edge Developer Tools for Visual Studio: Network tool of DevTools.](../../media/2022/03/devtools-extension-visual-studio-network.png)
+
+The **Inspect tool** (![Inspect tool icon.](../../media/2022/03/v-s-edge-devtools-inspect-tool-icon.png)) and **Toggle screencast** (![Toggle screencast icon.](../../media/2022/03/v-s-edge-devtools-toggle-screencast-icon.png)) are available, and the **More Tools** (![More Tools icon.](../../media/2022/03/more-tools-v-s-icon.png)) menu offers the [Issues](../../../issues/index.md), [Network conditions](../../../network-conditions/network-conditions-tool.md), and [Network request blocking](../../../network-request-blocking/network-request-blocking-tool.md) tools.
+
+See also:
+
+* [Edge Developer Tools for Visual Studio (Preview)](https://devblogs.microsoft.com/visualstudio/?p=237066&preview=1&_ppp=7aa7aef54f), at Visual Studio DevBlogs.
+* [Edge DevTools extension for Visual Studio](../../../../visual-studio/index.md#edge-devtools-extension-for-visual-studio) in _Visual Studio for web development_.
```

Comment:
I don't really get why this is italicized. If it's in our repo, then I'd recommend just doing:
```suggestion
* [Edge DevTools extension for Visual Studio](../../../../visual-studio/index.md#edge-devtools-extension-for-visual-studio).
```

Reply:
I will remove it but it is there for solid reason: because w/o it, the Content Developer (writer) can't tell where the link goes, while looking at .md tagging view.  Prevents being lost in navigation.  Warns author/reader where they'll end up after traversing the link: not an article's top, but an h2 halfway down in a page, which is not visibly clear after arrival, until scroll to top of the destination page - quite disorienting.